lib/jxl/dec_external_image.cc

```diff
@@ -293,13 +293,17 @@ Status ConvertChannelsToExternal(const ImageF* channels[], size_t num_channels,
   const size_t bytes_per_pixel = num_channels * bytes_per_channel;
 
   std::vector<std::vector<uint8_t>> row_out_callback;
-  auto InitOutCallback = [&](size_t num_threads) {
-    if (out_callback) {
+  void* out_run_opaque = nullptr;
+  auto InitOutCallback = [&](size_t num_threads) -> Status {
+    if (out_init_callback && out_run_callback) {
```

Comment:
We very likely should assert that they are all null, or all non-null.

Passing a struct might help for that. (or a class with constructor + getters, but that gets more red tape-y)